Added support for background image

JDSlimz opened the pull request:
Custom background image support and css parameters to control background repeat and size can be added in config file.

JDSlimz committed:
Added support for background image

Custom background image support and css parameters to control background repeat and size can be added in config file.

jmarmorato commented on app/views/main/header.php:
```diff
@@ -14,5 +14,15 @@
     <link rel="stylesheet" href="/darkmode_style.css">
   <?php endif; ?>
 
+  <?php if($data["config"]["background_image"] != ""): ?>
```
Let's add isset() to the if statement, something more like: `if(isset($data["config"]["background_image"]) && $data["config"]["background_image"] != "")`

(On line 17 of app/views/main/header.php)

Reason:  If someone updates to this commit from a previous commit, they will get the following error:

> Notice: Undefined index: background_image in /var/www/Intranet-Home-Page/app/views/main/header.php on line 17

By adding the `isset()` check **before** actually accessing $data["config"]["background_image"] index, we can avoid throwing an error when encountering an older version of the config.

If we can fix that, this looks great and I'd say it's good to go!

Thank you for your interest in the project and your contribution to the open-source community!

JDSlimz replied:
Excellent catch! It has been added. I look forward to adding some more ideas I have to this project!

jmarmorato reviewed (changes requested):
Excellent idea - I just have one request to help prevent errors when people upgrade from version to version.  Otherwise, I think this is a great idea.

Thank you for your interest and contribution!

JDSlimz committed:
Add isset in header to prevent error.

Added an isset per recommendation to prevent error should the background image not exist in the config.

jmarmorato commented:
Looks good, I'm looking forward to seeing some new ideas!
